spec/enumerables_spec.rb

```diff
@@ -0,0 +1,172 @@
+require_relative '../main.rb'
+# rubocop: disable  Lint/Void
+describe Enumerable do
+  let(:normal_array) { [1, 2, 3, 4, 5] }
+  let(:words) { %w[ant bear cat] }
+  let(:salad) { [nil, true, 99] }
+  let(:numbers) { [1, 3.14, 2i] }
+  let(:empty_array) { [] }
+  let(:hash) { {} }
+  let(:hash2) { {} }
+
+  describe '#my_each' do
+    subject { normal_array.each { |i| i * i } }
+    it 'Block Given, Travels array' do
+      subject { normal_array.each { |i| i * i } }
```

Comment:
Line 15 seems to be a duplication of line 13. I tried running RSpec by removing line 15 and it works fine. kindly remove it.